uniflow-test/build.gradle

```diff
@@ -5,9 +5,10 @@ description = 'UniFlow a Kotlin unidrectional data flow framework'
 
 dependencies {
     api project(":uniflow-core")
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutines_version"
-    implementation "junit:junit:$junit_version"
-    implementation "io.mockk:mockk:$mockk_version"
+    api "org.jetbrains.kotlin:kotlin-test:$kotlin_version"
```

Comment:
I don't think that this needs to be a transitive dependency.

In the future you could take a look at https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin to see what dependencies might need being added, removed or changed.

Reply:
as we use in android-test packages, it's interesting to make it transitive for us ... not sure it changes anything in metadata